### ISSUE TITLE
[qt5cpp] Map is generated as JsonArray instead of JsonObject

### ISSUE THREAD

**Issue by etherealjoy** [swagger-api/swagger-codegen#7198]
##### Description

There is a small issue related to the generation of Maps, wherein the QJsonArray is used instead of QJsonObject .

``` c++
    QJsonArray exposed_portsJsonArray;
    for(auto keyval : exposed_ports->keys()){
	    QJsonObject exposed_ports_jobj;
	    toJsonValue(keyval, ((*exposed_ports)[keyval]), &exposed_ports_jobj, "DockerObject");
	    exposed_portsJsonArray.append(exposed_ports_jobj);
	}   
    obj->insert("ExposedPorts", exposed_portsJsonArray);
```
instead of
``` c++
        QJsonObject exposed_ports_jobj;
        for(auto keyval : exposed_ports->keys()){
            toJsonValue(keyval, ((*exposed_ports)[keyval]), &exposed_ports_jobj, "DockerObject");
        }   
        obj->insert("ExposedPorts", exposed_ports_jobj);
```

##### Swagger-codegen version
2.3.0

**Comment by etherealjoy** [comment]
Can be closed after merging #7204

**Comment by etherealjoy** [comment]
Fixed with #7204
